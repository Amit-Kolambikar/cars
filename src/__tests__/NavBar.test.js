import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import NavBar from "../components/navbar/index";



describe("<CarsListItem />", () => {
  describe("render()", () => {
    test("renders the component", () => {
      const wrapper = shallow(<NavBar/>);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});
