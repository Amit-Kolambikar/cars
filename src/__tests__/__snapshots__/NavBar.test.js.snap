// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CarsListItem /> render() renders the component 1`] = `
<ForwardRef
  bg="#fff"
  bsPrefix="navbar"
  className="navbar-custom"
  expand="lg"
  innerRef={null}
>
  <NavbarBrand
    as={[Function]}
    to="/"
  >
    <img
      alt="auto1"
      className="d-inline-block align-top"
      src="logo.png"
      width="200"
    />
  </NavbarBrand>
  <NavbarToggle
    aria-controls="basic-navbar-nav"
    label="Toggle navigation"
  />
  <NavbarCollapse
    className="justify-content-end"
  >
    <NavLink
      as={[Function]}
      disabled={false}
      href="#home"
    >
      Purchase
    </NavLink>
    <NavLink
      as={[Function]}
      disabled={false}
      href="#link"
    >
      My Orders
    </NavLink>
    <NavLink
      as={[Function]}
      disabled={false}
      href="#link"
    >
      Sell
    </NavLink>
  </NavbarCollapse>
</ForwardRef>
`;
